lacci/lib/shoes/widgets/edit_box.rb

```diff
@@ -2,9 +2,9 @@
 
 module Shoes
   class EditBox < Shoes::Widget
-    display_properties :text, :height, :width
+    display_properties :text, :height, :width, :margin_bottom
```

Comment:
Margin-bottom would normally be added by spacing, like we do in Slot (which does the same for Stack and Flow.) Is there a reason not to do that here?